forc-test/src/lib.rs

```diff
@@ -871,6 +912,34 @@ mod tests {
         }
     }
 
+    fn test_contract_test_results(
```

Comment:
Similarly, this could be one function `get_test_results(package_name: &str, test_filter: Option<TestFilter>)`